unit_tests/testMultigridMesh/testMultigridMesh.f90

```diff
@@ -0,0 +1,57 @@
+!++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
+!                          Futility Development Group                          !
+!                             All rights reserved.                             !
+!                                                                              !
+! Futility is a jointly-maintained, open-source project between the University !
+! of Michigan and Oak Ridge National Laboratory.  The copyright and license    !
+! can be found in LICENSE.txt in the head directory of this repository.        !
+!++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
+PROGRAM testMultigridMesh
+#include "UnitTest.h"
+  USE ISO_FORTRAN_ENV
+  USE UnitTest
+  USE IntrType
+  USE MultigridMesh
+
+  IMPLICIT NONE
+
+  CREATE_TEST('Test MultigridMesh')
+
+  REGISTER_SUBTEST('Test Clear',testClear)
+  REGISTER_SUBTEST('Test Init',testInit)
+
+  FINALIZE_TEST()
+!
+!===============================================================================
+  CONTAINS
+!
+!-------------------------------------------------------------------------------
+  SUBROUTINE testClear
+    TYPE(MultigridMeshStructureType) :: myMMeshes
+
+    myMMeshes%nLevels=5
```

Comment:
Should clear also clear the other attributes and should this be checked? e.g. interp matrix?

Reply:
The interpolation matrices are not stored on MultigridMesh.f90.  I added another clear routine to check the clearing on the mesh level rather than the full structure level.